quesma/backend_connectors/elasticsearch_backend_connector.go

```diff
@@ -41,6 +42,24 @@ func NewElasticsearchBackendConnector(cfg config.ElasticsearchConfiguration) *El
 	return conn
 }
 
+// NewElasticsearchBackendConnector2 is an alternative constructor which uses the generic database configuration object
+func NewElasticsearchBackendConnector2(cfg config.RelationalDbConfiguration) *ElasticsearchBackendConnector {
```

Comment:
I'm wondering if we really need 2 constructors. Maybe we can create 
```
config: config.ElasticsearchConfiguration{
			Url:      cfg.Url,
			User:     cfg.User,
			Password: cfg.Password,
		},
```
outside and pass it to first constructor?

Reply:
The separation comes from the fact that I'd like to eventually get rid of `config.ElasticsearchConfiguration` (and ideally all of the legacy config. So I thought it could be nicer in the long run to just kick out the second constructor with all its usages further down the road